proposed/simplecache-meta.md

```diff
@@ -0,0 +1,85 @@
+PSR-Simple-Cache Meta Document
+==============================
+
+1. Summary
+----------
+
+Caching is a common way to improve the performance of any project, and many
+libraries make use or could make use of it. Interoperability at this level
+means libraries can drop their own caching implementations and easily rely
+on the one given to them by the framework, or another dedicated cache
+library the user picked.
+
+
+2. Why Bother?
+--------------
+
+PSR-6 solves this problem already, but in a rather formal and verbose way for
+what the most simple use cases need. This simpler approach aims to build a
+standardized layer of simplicity on top of the existing PSR-6 interfaces.
+
+
+3. Scope
+--------
+
+### 3.1 Goals
+
+* A simple interface for basic cache operations.
+* Basic support for operations on multiple keys for performance (round-trip-time)
+  reasons.
+* Support for atomic counters which are essential to implement security features
```

Comment:
@Seldaek I think he means on the extension level. I agree that this should be piped directly to extension; I can't think of a single use-case where you want to have a method around that.

Do you have a valid use-case for this? 

> I don't know why we bother with PSRs at all

Not all PSRs are created equally.